src/lib/game/carGame.ts

```diff
@@ -50,6 +50,12 @@ export function disconnect() {
 
   disposeCarGame();
 }
+export function ready(ready: boolean) {
+  carGame?.clientSocket.socket.emit('player-ready', ready);
+}
```

Comment:
As with `sendUserInput`, it might be easier to keep this communication in the clientSocket, so that it is easier to find and adjust what data we send and through what pipeline.